--- a/core/global/plugin_loader.gd
+++ b/core/global/plugin_loader.gd
@@ -432,6 +432,6 @@ func _load_plugins() -> void:
 
 		# Load the plugin, this will also replace the existing instance of the resource.
 		# for an updated plugin.
-		if not ProjectSettings.load_resource_pack("/".join([PLUGINS_DIR, file_name])):
+		if not ProjectSettings.load_resource_pack("/".join([PLUGINS_DIR, file_name]), true):
 			logger.warn("%s failed to load" % file_name)
 
